### ISSUE TITLE
Labels lookup searches for labelId and not name - not good for custom labels

### ISSUE THREAD

**Issue by Carpela** [jhk753/gmail-ruby-api#22]
mail = Gmail::Message.get mailId

mail.modify(["Some Label"],[])

Gives a Label not found error, even if the label exists because it's searching against labelId not against name.

**Comment by jhk753** [comment]
Yes it definitely look at labelsId, not sure how you want to deal with this?

**Comment by Carpela** [comment]
I wonder if there's a way to look for the label name as that's obviosuly what you'll be more familiar with, then automatically process that without having to do too many API calls.
